### ISSUE TITLE
Reconstruction loss for Bernoulli distributed data doesn't use logits for more stable training

### ISSUE THREAD

**Issue by luis-armando-perez-rey** [luis-armando-perez-rey/lsbd-vae#8]
To increase training stability we should:
- Remove the use of sigmoid in the decoding neural networks
- Change the Bernoulli loss to use logits
